.github/workflows/integrationtests.yaml

```diff
@@ -113,6 +113,34 @@ jobs:
       - name: Daps Integration Tests
         run: ./gradlew -p extensions/iam/daps test -DincludeTags="DapsIntegrationTest"
 
+  End-To-End-Tests:
```

Comment:
the newly tagged tests are not in the published test report anymore

should we run them in the UT job instead?